tests/device_tests/cardano/test_sign_tx.py

```diff
@@ -66,7 +66,16 @@ def test_cardano_sign_tx(client, parameters, result):
             auxiliary_data=auxiliary_data,
         )
         assert response.tx_hash.hex() == result["tx_hash"]
-        assert response.serialized_tx.hex() == result["serialized_tx"]
+        assert [witness[1].hex() for witness in response.signatures] == result[
```

Comment:
I think this looks nicer as:
```Python
assert [signature.hex() for (_, signature) in response.signatures] == result[
```
Although I am not sure whether we shouldn't also include the paths in the test cases results and call the field `witnesses`. I think it would make sense since that is what trezorlib returns, but I am not sure if that's something we should be testing. But I am leaning more towards including the paths in the expected results anyways. What do you think?